config/routes.rb

```diff
@@ -2,6 +2,7 @@
   
    get 'personal_page/index'
    get 'fun_page/index'
+   get 'social_projects_page/index'
```

Comment:
I have ,except for one, index actions in the controllers, what is the benefit of using resource here instead of get '..'? Better style or sth else?